data/Templates/Hl7v2/Resource/_Observation.liquid

```diff
@@ -22,11 +24,19 @@
         {% if OBX.2.Value == "ST" or OBX.2.Value == "FT" or OBX.2.Value == "TX" -%}
         "valueString":"{{ OBX.5.Value }}",
         {% endif -%}
+        {% if OBX.2.Value == "VR" -%}
+        "valueString":"{{ OBX.5.1 }}-{{ OBX.5.2 }}",
+        {% endif -%}
+        {% if OBX.2.Value == "SN" -%}
+        "valueString":"{{ OBX.5.1 }} {{ OBX.5.2 }} {{ OBX.5.3 }}",
+        {% endif -%}
```

Comment:
Are we getting these changes by updating the mapping tables?

Reply:
Besides it seems some data are combined by hyphen while some others are combined by space. Are we expecting this to happen?

Reply:
These changes are updated manually. The automatically generated result is `"valueString":OBX-5.1+" "+OBX-5.2+" "+OBX-5.3`, which is an invalid expression. I manually changed it to the `"valueString":"{{ OBX.5.1 }} {{ OBX.5.2 }} {{ OBX.5.3 }}"`.
The joining characters(" " and "-") are inherited from mapping table.